src/storage/StorageServer.cpp

```diff
@@ -89,11 +90,14 @@ bool StorageServer::start() {
     workers_->start();
 
     // Meta client
+    meta::MetaClientOptions options;
+    options.localHost_ = localHost_;
+    options.inStoraged_ = true;
+    options.serviceName_ = "";
```

Comment:
why the service name is empty ?

Reply:
Ha,  it is compatible with current logic.  I did not handled it in this pr.